src/osd/OSD.cc

```diff
@@ -5834,8 +5834,40 @@ bool OSD::scrub_random_backoff()
   return false;
 }
 
+bool OSD::scrub_time_permit()
```

Comment:
I was thinking we should test both osd_scrub_begin_hour < and > osd_scrub_end_hour.  So I'd prefer we be able to inject a test time so we can do the following type of test cases.

current time  osd_scrub_begin_hour    osd_scrub_end_hour    result
01:00                  22                                        03                         true
20:00                  22                                         03                        false
04:00                 22                                          03                        false
02:00                 01                                          03                        true
00:30                 01                                          03                        false
08:00                 01                                          03                        false